src/spring-cloud/azext_spring_cloud/application_configuration_service.py

```diff
@@ -0,0 +1,224 @@
+    _validate_acs_settings(client, resource_group, service, acs_settings)
+
+    logger.warning("[2/2] Updating Application Configuration Service settings repo, (this operation can take a while to complete)")
+    return sdk_no_wait(no_wait, client.configuration_services.begin_create_or_update, resource_group, service, acs_resource)
+
+
+def application_configuration_service_git_remove(cmd, client, service, resource_group, name, no_wait=False):
+    acs_resource = client.configuration_services.get(resource_group, service)
+    acs_settings = acs_resource.properties.settings
+    if not acs_settings or not acs_settings.git_property or not acs_settings.git_property.repositories:
+        raise CLIError("Repo '{}' not found.".format(name))
+
+    git_property = acs_settings.git_property
+    repository = [repo for repo in git_property.repositories if repo.name == name]
+    if not repository:
+        raise CLIError("Repo '{}' not found.".format(name))
+
+    git_property.repositories.remove(repository[0])
+
+    acs_settings = models.ConfigurationServiceSettings(git_property=git_property)
+    properties = models.ConfigurationServiceProperties(settings=acs_settings)
+
+    logger.warning("[1/2] Validating Application Configuration Service settings")
+    _validate_acs_settings(client, resource_group, service, acs_settings)
+
+    logger.warning("[2/2] Deleting Application Configuration Service settings repo, (this operation can take a while to complete)")
+    acs_resource = models.ConfigurationServiceResource(properties=properties)
+    return sdk_no_wait(no_wait, client.configuration_services.begin_create_or_update, resource_group, service, acs_resource)
+
+
+def application_configuration_service_git_list(cmd, client, service, resource_group):
+    acs_resource = client.configuration_services.get(resource_group, service)
+    acs_settings = acs_resource.properties.settings
+
+    if not acs_settings or not acs_settings.git_property or not acs_settings.git_property.repositories:
+        raise CLIError("Repos not found.")
+
+    return acs_settings.git_property.repositories
+
+
+def application_configuration_service_bind(cmd, client, service, resource_group, app):
+    if is_enterprise_tier(cmd, resource_group, service):
+        _acs_bind_or_unbind_app(cmd, client, service, resource_group, app, True)
+    else:
+        raise CLIError("Application Configuration Service is only supported in Enterprise tier.")
+
+
+def application_configuration_service_unbind(cmd, client, service, resource_group, app):
+    if is_enterprise_tier(cmd, resource_group, service):
+        _acs_bind_or_unbind_app(cmd, client, service, resource_group, app, False)
+    else:
+        raise CLIError("Application Configuration Service is only supported in Enterprise tier.")
+
+
+def _acs_bind_or_unbind_app(cmd, client, service, resource_group, app_name, enabled):
+    app = app_get_enterprise(cmd, client, resource_group, service, app_name)

```

Comment:
`app_get_enterprise` also retrieves active deployment, I dont think this bind/unbind needs the deployment. How about using SDK client to directly get app?